app/serializers/api/adjustment_serializer.rb

```diff
@@ -0,0 +1,8 @@
+module Api
+  class AdjustmentSerializer < ActiveModel::Serializer
+    attributes :id, :amount, :label, :eligible
+    attributes :source_type, :source_id
+    attributes :adjustable_type, :adjustable_id
+    attributes :originator_type, :originator_id
```

Comment:
you can split it into multiple lines without having to call `attributes` many times. IMO that will highlight the fact that they all are attributes of the serializer and that will still be readable. Well, maybe you did it for consitency.

Reply:
I did it for consistency with the orderserializer but now I see most of them dont have this.
so I fixed all serializers and removed duplicate attributes. new commit.